// Author: Allen Porter <allen@thebends.org>

package proto;

message MountRequest {
  required string fs_id = 1; 
  required string volume = 2;
}

message MountResponse {
}

service MountService {
  rpc Mount (MountRequest) returns (MountResponse);
}
